app/uk/gov/hmrc/cataloguefrontend/leakdetection/LeakDetectionService.scala

```diff
@@ -94,9 +96,33 @@ class LeakDetectionService @Inject() (
     leakDetectionConnector.leakDetectionReport(repository, branch)
 
   def reportLeaks(reportId: String)(implicit hc: HeaderCarrier): Future[Seq[LeakDetectionLeaksByRule]] =
-    leakDetectionConnector.leakDetectionLeaks(reportId).map(_.groupBy(l => l.ruleId)
-      .map {
-        case (ruleId, leaks) =>
+    leakDetectionConnector.leakDetectionLeaks(reportId)
+      .map(_
+        .filterNot(_.excluded)
+        .groupBy(_.ruleId)
+        .map {case (ruleId, leaks) => mapLeak(ruleId, leaks)}
+        .toSeq
+        .sortBy(_.leaks.length)
```

Comment:
Not introduced in this change, but wonder if it might be worth defining an explicit `Ordering` for `LeakDetectionLeaksByRule` and just `.sorted` at the call site?